src/main/app/jurisdiction/moreDetailsContent.ts

```diff
@@ -4,6 +4,10 @@ export const enHabitualResident = `If your lives are mainly based in England or
   This may include working, owning property, having children in school, and your main family life taking place in England or Wales.<br><br>
   The examples above aren’t a complete list of what makes up habitual residence, and just because some of them apply to you doesn’t mean you’re habitually resident. If you’re not sure, you should get legal advice.`;
 
+export const enHabitualResidentCy = `Os ydych chi’n treulio’r rhan fwyaf o’ch amser yng Nghymru neu Loegr, rydych yn ‘preswylio’n arferol’ yno yn ôl y gyfraith.<br><br>
+  Gall hyn gynnwys gweithio, bod yn berchen ar eiddo, bod â phlant mewn ysgol, a bod eich prif fywyd teuluol yng Nghymru neu Loegr.<br><br>
+  Nid yw’r enghreifftiau uchod yn rhestr gyflawn o’r amgylchiadau sy’n diffinio preswylfa arferol. Nid yw’r ffaith bod rhai ohonynt yn berthnasol i chi o reidrwydd yn golygu eich bod yn preswylio’n arferol yn rhywle. Os nad ydych yn siŵr, dylech gael cyngor cyfreithiol.`;
```

Comment:
the en at the start of enHabitualResident indicates English, so these can be called cyHabitualResident, cyDomicile etc...